tests/libvmi/OWNERS

```diff
@@ -1,3 +1,5 @@
 # See the OWNERS docs at https://go.k8s.io/owners
 approvers:
   - EdDev
+labels:
+  - sig/test
```

Comment:
I would suggest to add this on the `tests/OWNERS`.

For now, it can be here as well, we do not use it on production for now.

Reply:
Fixed. Do you happen to know if lbales are inheirted from tests/?

Reply:
By default all owner files are inherited. A special flag needs to be added to drop the inheritance.